index.js

```diff
@@ -83,7 +83,7 @@ function uglifyify(file, opts) {
 
       map.setProperty('sources', [path.basename(file)])
       map.setProperty('sourcesContent', matched
-        ? opts.inSourceMap.sourcesContent
+        ? opts.sourceMap.sourcesContent
```

Comment:
For what i could see when i was debugging `sourceMap.sourcesContent` is the right field, `sourceMap.content` has the entire source map json